runner/src/server/config.ts

```diff
@@ -170,5 +171,12 @@ export function buildConfig() {
 }
 
 const config = buildConfig();
-
+// TODO: migrate all environment variables to runner/config/default.json
+config.safelist = nodeConfig.get("safelist");
+config.initialisedSessionTimeout = nodeConfig.get("initialisedSessionTimeout"); //default 28 days
+if (!nodeConfig.has("initialisedSessionKey")) {
+  config.initialisedSessionKey = nanoid(16);
+} else {
+  config.initialisedSessionKey = nodeConfig.get("initialisedSessionKey");
+}
```

Comment:
A couple of things here - you could put all the env vars in the `buildConfig` function into the custom-environment-variables.json file and let node config take care of it all. But if that's too much for this change, these steps here copying in further config should probably be done in the `buildConfig` function, before the config is validated.